docs/messaging.rst

```diff
@@ -1313,6 +1334,86 @@ of the debugger to the ``global`` scope to inspect it after debug session.
 
 .. versionadded:: 5.5
 
+.. _kernel_subshells:
+
+Kernel subshells
+----------------
+
+Kernel subshells are separate threads of execution within the same kernel process that
+were introduced by
+`Jupyter Enhancement Proposal 91 <https://github.com/jupyter/enhancement-proposals/pull/91>`_.
+Kernels supporting subshells must return ``'supported_features'=['kernel subshells']``
```

Comment:
Maybe say that kernels supporting subshells must include `'kernel subshells'` in `'supported_features'`, since there could be other features in there?

Reply:
Done.